### PR TITLE
Add docs for workload cluster support for tinkerbell

### PR DESCRIPTION
**Pull request by panktishah26**
*Issue #, if available:*

*Description of changes:*

*Testing (if applicable):*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/content/en/docs/getting-started/production-environment/baremetal-getstarted.md [hunk 1]
```diff
@@ -108,6 +111,71 @@ Follow these steps to create an EKS Anywhere cluster.
...
+
+1. Generate a workload cluster config:
+   ```bash
+   CLUSTER_NAME=w01
+   eksctl anywhere generate clusterconfig $CLUSTER_NAME \
+      --provider tinkerbell > eksa-w01-cluster.yaml
+   ```
+
+   Refer to the initial config described earlier for the required and optional settings.
+   Ensure workload cluster object names (`Cluster`, `TinkerbellDatacenterConfig`, `TinkerbellMachineConfig`, etc.) are distinct from management cluster object names. Be sure to set the `managementCluster` field to identify the name of the management cluster. Keep the tinkerbellIP of workload cluster same as tinkerbellIP of the management cluster.
```
Comment (chrisnegus) [inline docs/content/en/docs/getting-started/production-environment/baremetal-getstarted.md:126]:
```suggestion
   Ensure workload cluster object names (`Cluster`, `TinkerbellDatacenterConfig`, `TinkerbellMachineConfig`, etc.) are distinct from management cluster object names. Be sure to set the `managementCluster` field to identify the name of the management cluster. Keep the tinkerbellIP of workload cluster the same as tinkerbellIP of the management cluster.
```

**Review by chrisnegus — COMMENTED** [review]
Looks great. One very small wording suggestion.

**Review by TerryHowe — APPROVED** [review]
/lgtm

**Comment by TerryHowe** [comment]
This should probably be cherry picked???

**Pull request merged**